packages/core/scripts/severless-cloud-run/CloudRunSpoke.js

```diff
@@ -96,7 +96,7 @@ function _stripExecOutput(output) {
   return output
     .replace(/\r?\n|\r/g, "") // Remove escaped new line chars
     .replace(/\s\s+/g, " ") // Remove tabbed chars
-    .replace(/\"/g, ""); // Remove escaped quotes
+    .replace(/"/g, ""); // Remove escaped quotes
```

Comment:
@chrismaree FYI take a second look at this

Reply:
Yes, +1. It said the escape was unnecessary. I went with its suggestion, but we should put another set of eyes on it rather than blindly trusting eslint.